x-pack/lib/filters/geoip/database_manager.rb

```diff
@@ -106,16 +116,23 @@ def execute_download_job
             logger.info("geoip plugin will use database #{new_database_path}",
                         :database_type => db_type, :pipeline_ids => ids) unless ids.empty?
           end
+
+          success_cnt += 1
         end
       end
 
       updated_types = updated_db.map { |database_type, valid_download, dirname, new_database_path| database_type }
-      (DB_TYPES - updated_types).each { |unchange_type| @metadata.update_timestamp(unchange_type) }
+      (DB_TYPES - updated_types).each do |unchange_type|
+        @metadata.update_timestamp(unchange_type)
+        success_cnt += 1
+      end
     rescue => e
       logger.error(e.message, error_details(e, logger))
     ensure
       check_age
       clean_up_database
+      set_download_metric(success_cnt)
```

Comment:
Here we could also use `update_download_status` so in the flow of download a DB becomes more evident the status change points.